api/main_endpoints/routes/User.js

```diff
@@ -199,6 +207,73 @@ router.post('/getPagesPrintedCount', (req, res) => {
   });
 });
 
+// Verifies the users session if they have an active jwtToken.
+// Used on the inital load of root '/'
+// Returns the name and accesslevel of the user w/ the given access token
+router.post('/verify', function(req, res) {
```

Comment:
can you remove lines `210 - 225`? we already have this [here](https://github.com/SCE-Development/Core-v4/blob/e465da89e0719bd60f5c009267e017242d9ceaff/api/main_endpoints/routes/Auth.js#L151)

Reply:
I believe that was already there

Reply:
then why is it green in this pr